server/services/forms/schemas/document-add-next-case.js

```diff
@@ -5,6 +5,20 @@ const { MIN_ALLOWABLE_YEAR } = require('../../../libs/dateHelpers');
 
 module.exports = (options) => {
     const fromCaseUUID = options?.data?.from ? options.data.from : '';
+    const documentRequired = options.customConfig ? options.customConfig.documentRequired : true;
```

Comment:
Unsure from a readability point of view but you could do: 

`options?.config?.documentRequired || true`

Reply:
I don't think that will work because when `documentRequired` is set to false that will output true

Reply:
Thats true - look into the null coelescing operator `??`, that only checks for undefined or null.